draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -1252,10 +1288,40 @@ uCDN-A                         dCDN-B                         dCDN-C
    Value: An array of Error.v2 Descriptions, as defined in <xref target="error-v2-description"/>. 
    An empty array is allowed and is equivalent to omitting "errors" from the object.
    <vspace blankLines="1"/>
-   Mandatory: No. dCDN SHOULD include this attribute in the trigger resource representations it sends when the trigger is in "failed" state.
+   Mandatory: No. dCDN SHOULD include this attribute in the trigger resource representations it sends 
+   when the trigger is in "failed" state.
    Mandatory: No.
    </t>
 
+   <t hangText="Name: total-objects-count">
+   <vspace blankLines="1"/>
+   Description: Number of objects affected by the trigger, for example number of objects purged, invalidated or 
+   prepositioned as a result of trigger processing.  It is RECOMMENDED that dCDN maintains this attribtue as a cumulative
+   counter across all of its nodes, without de-duplicating same objects processed in multiple nodes. 
+   The primary purpose of this attribute is to help uCDN identify abnormal trigger processing results, that affected no
```

Comment:
Instead of "that affected no objects or unexpectedly low number of objects", maybe reword this to, "e.g. a purge that impacted a lower number of objects than expected."  That also handles the 0 objects case.

Reply:
rewrote to say "e.g. purge or preposition trigger that impacted a lower number of objects than expected."